dds_web/config.py

```diff
@@ -74,3 +74,7 @@ class Config(object):
     SUPERADMIN_EMAIL = os.environ.get("DDS_SUPERADMIN_EMAIL", "superadmin@example.com")
 
     REVERSE_PROXY = False  # Behind a reverse proxy, use X_Forwarded-For to get the ip
+
+    # Logging setup
```

Comment:
```suggestion
    # Logging setup; 0 means no log rotation
```

Reply:
Ah ok, thanks for the clarification!

Reply:
https://docs.python.org/3/library/logging.handlers.html#rotatingfilehandler